src/seedu/addressbook/commands/ExitCommand.java

```diff
@@ -13,6 +17,10 @@
 
     @Override
     public CommandResult execute() {
+        Iterator<Tagging> tagHistory_itr = Tagging.tagHistory.iterator();
```

Comment:
Coding standard violation! One more more of these problems detected: 
1. extra/missing spaces. 
2. brace style problem. 
3. missing braces.
4. name doesn't follow naming convention